templates/body1_server.xml.tomcat7.erb

```diff
@@ -29,9 +29,13 @@
   <Listener className="org.apache.catalina.security.SecurityListener" />
   -->
   <!--APR library loader. Documentation at /docs/apr.html -->
+<% if ! @apr_listener -%>
```

Comment:
Actually, it might just be easier to not include the commented out setting by default, and only add it when `$apt_listener` is set…

Reply:
I can do that as well.  I only left the commented-out section so that the configuration wouldn't change existing configurations, causing Tomcat restarts.  If you would prefer just leaving out the commented-out sections, I can update the branch.

Reply:
@raphink I changed the template per your suggestion.  Let me know if this looks correct now.

Reply:
Also, this will change the default on Tomcat 6 which was to enable the APR listener by default.  We could also change the default in this change to **enabled**, which would change the default for Tomcats 7 and 8.  In either case, it would probably be something to call out in the change log as it would potentially change how Tomcat was configured when updating the module.